pkg/sql/delegate/show_completions_test.go

```diff
@@ -87,6 +87,16 @@ func TestCompletions(t *testing.T) {
 			expectedCompletions: []string{"SELECT"},
 			offset:              12,
 		},
+		{
+			stmt:                "😋😋😋 😋😋😋",
+			expectedCompletions: []string{},
+			offset:              25,
```

Comment:
Can this be updated to use offset 30 such that the test covers the example in the issue description.